packages/app-store/routing-forms/lib/getAttributes.ts

```diff
@@ -53,37 +53,41 @@ async function getAttributeToUserWithMembershipAndAttributesForTeam({ teamId }:
 }
 
 async function getAttributesAssignedToMembersOfTeam({ teamId }: { teamId: number }) {
-  const log = logger.getSubLogger({ prefix: ["getAttributeToUserWithMembershipAndAttributes"] });
+  const membersOfTeam = await prisma.membership.findMany({
```

Comment:
Generally when we split queries we’ve been awaiting them using await Promise.all. Want to do that here?

Reply:
Don't think we can do that here though, each query depends on the output of the previous one.

Reply:
Never mind. Didn’t see that. Forgive me.